### ISSUE TITLE
HTTP logging is set as INFO?

### ISSUE THREAD

**Issue by andrenatal** [qdrant/qdrant-client#242]
It seems the http logging in the python library is set as INFO? Wouldn't be better if that's set to DEBUG so it doesn't conflict with actual INFO messages?

**Comment by tamaravasey** [comment]
Yes this would be helpful to drop down to DEBUG, especially when calling scroll() often due to the result pagination.

**Comment by generall** [comment]
You can disable it by specifying `INFO,actix_web=WARN`

**Comment by ambrustorok** [comment]
Is this still relevant @generall? `logging.getLogger("actix_web").setLevel(logging.WARNING)` does not work for me, I still get the messages in the log.
